app/services/appropriate_bodies/process_batch/claim.rb

```diff
@@ -0,0 +1,76 @@
+module AppropriateBodies
+  module ProcessBatch
+    # Claim
+    class Claim < Base
+      # @return [CSV::Table]
+      def process!
+        pending_induction_submission_batch.rows.each do |row|
+          @row = row
+          @pending_induction_submission = sparse_pending_induction_submission
+
+          claim!
+        rescue Errors::TeacherHasActiveInductionPeriodWithCurrentAB
+          pending_induction_submission.update(error_message: "Already claimed by your appropriate body")
+          next
+        rescue Errors::TeacherHasActiveInductionPeriodWithAnotherAB
+          pending_induction_submission.update(error_message: "Already claimed by another appropriate body")
+          next
+        rescue ::TRS::Errors::TeacherNotFound
+          pending_induction_submission.update(error_message: "Not found in TRS")
+          next
+        rescue ::TRS::Errors::ProhibitedFromTeaching
+          pending_induction_submission.update(error_message: "Prohibited from teaching")
+          next
+        rescue ::TRS::Errors::QTSNotAwarded
+          pending_induction_submission.update(error_message: "QTS not awarded")
+          next
+        rescue StandardError => e
+          pending_induction_submission.update(error_message: e.message)
```

Comment:
Similar to my other comment, could these 4 could be extracted into a `reject_with_error(message)` to keep things DRY-er?

```ruby
def reject_with_error(message)
  pending_induction_submission.update(error_message: message)
end
```